python_modules/dagster/dagster/_core/pipes/context.py

```diff
@@ -292,6 +295,22 @@ def get_bootstrap_env_vars(self) -> Dict[str, str]:
             for param_name, param_value in self.get_bootstrap_params().items()
         }
 
+    @public
+    def get_bootstrap_cli_arguments(self) -> Dict[str, str]:
```

Comment:
```suggestion
    def get_bootstrap_cli_arguments(self) -> Mapping[str, str]:
```

We generally prefer `Mapping` over `Dict`.

Reply:
ok, I also changed other similar methods annotations - that's fine?